tests/non-ui/config.js

```diff
@@ -0,0 +1,50 @@
+const request = require('request-promise-native');
```

Comment:
I like that this is pulled out into it's own module. Take the time to update `tests/conf.js` so we don't have duplicate code.

Reply:
good point - done